### PR TITLE
Layout files section

### PR DESCRIPTION
**Pull request by zero-24**
@Bakual as noted here is a proposal for the layout files section :smile:

### CODE REVIEW

File: manual/en-US/coding-standards/chapters/php.md [hunk 1]
```diff
@@ -131,6 +131,25 @@ switch ($value)
 }
 ```
 
+## Mixed language usage (e.g. at the layout files)
+
+For layout files and all files where we use a mix of PHP and HTML (all PHP files in the `view/tmpl` and `layout` folder) we wrapped additionally every line into a `<?php ... ?>` block and use the alternative syntax for control structures.
```
Comment (brianteeman) [inline manual/en-US/coding-standards/chapters/php.md]: We additionally wrap

File: manual/en-US/coding-standards/chapters/php.md [hunk 2]
```diff
@@ -131,6 +131,25 @@ switch ($value)
...
+Reason is that this way is supposed to be easier to understand by frontend guys and regular users. You can also easier move blocks around without creating fatal errors due to missing `<?php ... ?>` tags and the like.
```
Comment (brianteeman) [inline manual/en-US/coding-standards/chapters/php.md]: Replace first sentence with

This should make the code easier to read.

Comment (mbabker) [inline manual/en-US/coding-standards/chapters/php.md]: I would try to find a better way to word this.  I've read this sentence a couple of times today and each time the first thing that I react with is "man this feels like we're talking down on non-developers".

Comment (zero-24) [inline manual/en-US/coding-standards/chapters/php.md]: @mbabker also with the new wording by @brianteeman ?

`This should make the code easier to read.`

Comment (mbabker) [inline manual/en-US/coding-standards/chapters/php.md]: That reads better (I guess we were posting at the same time, didn't see his comment before).

File: manual/en-US/coding-standards/chapters/php.md [hunk 3]
```diff
@@ -131,6 +131,25 @@ switch ($value)
 }
 ```
 
+## Mixed language usage (e.g. at the layout files)
+
+For layout files and all files where we use a mix of PHP and HTML (all PHP files in the `view/tmpl` and `layout` folder) we additionally wrap every line into a `<?php ... ?>` block and use the alternative syntax for control structures.
+This should make the code easier to read. You can also easier move blocks around without creating fatal errors due to missing `<?php ... ?>` tags and the like.
```
Comment (brianteeman) [inline manual/en-US/coding-standards/chapters/php.md]: ... to read and make it easier to move blocks ...

Comment (brianteeman) [inline manual/en-US/coding-standards/chapters/php.md]: Don't put and the like. It's not needed a full stop is OK here

Comment (zero-24) [inline manual/en-US/coding-standards/chapters/php.md]: fixed both with: https://github.com/zero-24/coding-standards/commit/b26f6dd36ed3f30b8fb7b16a8ce2ce1d09500f6b and https://github.com/zero-24/coding-standards/commit/a28bcc56646a7d6a9eb63112dc703232b7fe7eeb

**Pull request merged**